spring-cloud-consul-discovery/src/main/java/org/springframework/cloud/consul/discovery/ConsulServer.java

```diff
@@ -33,11 +34,15 @@
 
 	private final HealthService service;
 
+	private final Set<Check.CheckStatus> statusConsideredAsHealthy;
+
 	private final Map<String, String> metadata;
 
-	public ConsulServer(final HealthService healthService) {
+	public ConsulServer(final HealthService healthService,
```

Comment:
We need to maintain backwards compatibility so we need to leave this constructor as is (perhaps mark it as deprecated) and add a new one.